homeassistant/components/qwikswitch.py

```diff
@@ -4,18 +4,21 @@
 For more details about this component, please refer to the documentation at
 https://home-assistant.io/components/qwikswitch/
 """
+import asyncio
 import logging
 
 import voluptuous as vol
 
 from homeassistant.const import (
     EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP, CONF_URL)
+from homeassistant.core import callback
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
 from homeassistant.helpers.discovery import load_platform
 from homeassistant.components.light import (
     ATTR_BRIGHTNESS, SUPPORT_BRIGHTNESS, Light)
```

Comment:
It's a bad pattern to import the `Light` entity class into the component. The platform specific entity classes should stay in the platforms.

If all platforms have the same logic regarding entity instantiation and you want to be DRY, you can eg instead create a common function used in `setup_platform` that takes the platform entity class as an argument to instantiate the entities. This function can be defined here in the component and be imported into the platforms.